processes/local_development/seed_test_data.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+from uuid import uuid4
+from mappings.base_mapping import BaseMapping
+from datetime import datetime, timezone
+from model import Record
+from constants.get_constants import get_constants
+from ..core import CoreProcess
+from logger import create_log
+from managers import RedisManager
+from processes import CatalogProcess, ClassifyProcess, ClusterProcess, HathiTrustProcess
+
+logger = create_log(__name__)
+
+class TestMapping(BaseMapping):
+    def __init__(self, record):
+        self.record = record
+
+    def createMapping(self):
+         pass
+
+class testSeedLocalDataProcess(CoreProcess):
```

Comment:
Let's call this SeedTestDataProcess. 

We are moving away from inheriting from the CoreProcess - recommend removing it!